RecoTracker/IterativeTracking/python/DisplacedGeneralStep_cff.py

```diff
@@ -0,0 +1,226 @@
+    SeedComparitorPSet = dict(
+        ComponentName = 'CombinedSeedComparitor',
+        mode = cms.string("and"),
+        comparitors = cms.VPSet(
+            cms.PSet(# FIXME: is this defined in any cfi that could be imported instead of copy-paste?
+                ComponentName = cms.string('PixelClusterShapeSeedComparitor'),
+                FilterAtHelixStage = cms.bool(True),
+                FilterPixelHits = cms.bool(False),
+                FilterStripHits = cms.bool(True),
+                ClusterShapeHitFilterName = cms.string('displacedGeneralStepClusterShapeHitFilter'),
+                ClusterShapeCacheSrc = cms.InputTag("siPixelClusterShapeCache") # not really needed here since FilterPixelHits=False
+            ), 
+            _StripSubClusterShapeSeedFilter.clone()
+        )
+    )
+)
+
+
+
+#----------------------------------------- QUALITY CUTS DURING TRACK BUILDING
+import TrackingTools.TrajectoryFiltering.TrajectoryFilter_cff as _TrajectoryFilter_cff
+_displacedGeneralStepTrajectoryFilterBase = _TrajectoryFilter_cff.CkfBaseTrajectoryFilter_block.clone(
+    maxLostHits = 1,
+    minimumNumberOfHits = 4,
+    minPt = 1,
+)
+
+displacedGeneralStepTrajectoryFilter = _displacedGeneralStepTrajectoryFilterBase.clone(
+    seedPairPenalty = 1,
+)
+
+
+displacedGeneralStepTrajectoryFilterInOut = displacedGeneralStepTrajectoryFilter.clone()
+
+import RecoTracker.MeasurementDet.Chi2ChargeMeasurementEstimator_cfi
+displacedGeneralStepChi2Est = RecoTracker.MeasurementDet.Chi2ChargeMeasurementEstimator_cfi.Chi2ChargeMeasurementEstimator.clone(
+    ComponentName = 'displacedGeneralStepChi2Est',
+    nSigma = 3.0,
+    MaxChi2 = 10.0,
+    clusterChargeCut = cms.PSet(refToPSet_ = cms.string('SiStripClusterChargeCutTight')),
+)
+
+
+
+#----------------------------------------- TRACK BUILDING
+import RecoTracker.CkfPattern.GroupedCkfTrajectoryBuilder_cfi
+displacedGeneralStepTrajectoryBuilder = RecoTracker.CkfPattern.GroupedCkfTrajectoryBuilder_cfi.GroupedCkfTrajectoryBuilder.clone(
+    MeasurementTrackerName = '',
+    trajectoryFilter = cms.PSet(refToPSet_ = cms.string('displacedGeneralStepTrajectoryFilter')),
+    inOutTrajectoryFilter = cms.PSet(refToPSet_ = cms.string('displacedGeneralStepTrajectoryFilterInOut')),
+    useSameTrajFilter = False,
+    minNrOfHitsForRebuild = 4,
+    maxCand = 2,
+    estimator = 'displacedGeneralStepChi2Est'
+    )
+
+
+
+#----------------------------------------- MAKING OF TRACK CANDIDATES
+import RecoTracker.CkfPattern.CkfTrackCandidates_cfi
+displacedGeneralStepTrackCandidates = RecoTracker.CkfPattern.CkfTrackCandidates_cfi.ckfTrackCandidates.clone(
+    src = 'displacedGeneralStepSeeds',
+    TrajectoryCleaner = 'displacedGeneralStepTrajectoryCleanerBySharedHits',
+    ### these two parameters are relevant only for the CachingSeedCleanerBySharedInput
+    numHitsForSeedCleaner = cms.int32(50),
+    onlyPixelHitsForSeedCleaner = cms.bool(True),
```

Comment:
```suggestion
    onlyPixelHitsForSeedCleaner = cms.bool(False),
```
?

While making an unrelated inspection of seed cleaning settings, I realized that tobTec has this requirement for no particularly clear reason

Reply:
from https://github.com/cms-sw/cmssw/pull/32036#discussion_r560191319

>     In the context of optimising the new iteration I already tried to change that to see if there is an obvious reason as why we would set it to false, but it didn't give any significant change.

this was recently checked/updated in #32579.
So, I suggest to update here now as well.